api/tests/bootstrap.php

```diff
@@ -4,6 +4,8 @@
 
 require dirname(__DIR__).'/vendor/autoload.php';
 
+DG\BypassFinals::enable(); // allows mocking of final classes
```

Comment:
why do we need this?

Reply:
`ApiPlatform\Doctrine\Orm\Filter\SearchFilter` is now newly a final class. In `RelatedCollectionLinkNormalizerTest` we create a test mock for a SearchFilter. However, creating a mock for a final class throws an error.

This is one of various solutions i found on:
https://tomasvotruba.com/blog/2019/03/28/how-to-mock-final-classes-in-phpunit/